Items/04_00_WeaponAttachments.txt

```diff
@@ -601,6 +601,23 @@ biologic targets and armor.
 by each shot by 20 points, but if the round hits the health pool of a cybernetic
 target, it deals an extra 45 damage.
 
+== Flechette Shells ($2.5 x mag cost) ==
+
+	Flechette (Fla-ch-ette) shells are a shotgun only ammo type, which contain 
+many flight sustaining, small, deadly, sharp pointed bits of a titanium alloy. 
+They are quite effective at rendering flesh and circuitry to a greater weakened state,
+and can pierce through materials to ensure the target is reached. These particular type
+of pointed bits are great at travelling greater distances, even while inside a target,
+having greater chance to hit something vital, causing great pain/malfunctions. 
+
+When using Flechette shells, increase damage 40% when shots pierce or miss armor. APL 
+of all guns ammo is used in is increased by 1. All range brackets gain a 5m increase,

```

Comment:
nonblocking: Consider rephrasing "all guns ammo is used in is increased by." It works, but I think we can make it a little easier to read.